sdk/storage/azure-storage-file-share/azure/storage/fileshare/aio/_share_client_async.py

```diff
@@ -283,9 +293,17 @@ async def delete_share(
         """
         access_conditions = get_access_conditions(kwargs.pop('lease', None))
         timeout = kwargs.pop('timeout', None)
+        include_snapshots = kwargs.pop('include_snapshots', None)
         delete_include = None
         if delete_snapshots:
+            warn("The `delete_snapshots` parameter is being deprecated. "
```

Comment:
we don't need to warn users. Some existing users are using delete_snapshot, they would complain about this warning.

Reply:
The warning wouldn't cause any breaking changes and I feel moving forward its better to adopt the new enum `include_snapshots`. I think a deprecation warn is a good idea for future proofing.